UNRELEASED.md

```diff
@@ -6,6 +6,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+-`TextField` no longer uses `componentWillReceiveProps` ([#628](https://github.com/Shopify/polaris-react/pull/628))
+
```

Comment:
```suggestion
```

Reply:
Prettier added this line, let me restart vscode code to check if it was an oddity.

Reply:
Seems prettier wants this line